testsuite/certificates/__init__.py

```diff
@@ -15,6 +15,7 @@ class CFSSLException(Exception):
 @dataclasses.dataclass
 class CertInfo:
     """Certificate configuration details"""
+
```

Comment:
This type of change is welcome because it makes code more readable by my opinion.